app/src/main/java/com/waseefakhtar/doseapp/feature/addmedication/AddMedicationRoute.kt

```diff
@@ -65,8 +67,10 @@ fun AddMedicationRoute(
     onBackClicked: () -> Unit,
     navigateToMedicationConfirm: (Medication) -> Unit,
     modifier: Modifier = Modifier,
-    // viewModel: CalendarViewModel = hiltViewModel()
+    viewModel: MedicationAddViewModel = hiltViewModel()
```

Comment:
As far as what I've currently planned, we don't need a ViewModel for AddMedication. We simply pass the gathered info to `MedicationConfirm` using a Medication object.

Reply:
Gotcha, thanks for the insight, I will update this line in the next push